lib/CLI.js

```diff
@@ -367,16 +419,32 @@ ${packageData.name} v${packageData.version} \
 
   exitWithStatus(error, stats) {
     if (error) {
-      if (error.message) { logger.error(error.message); }
+      if (error.message) {
+        process.stdout.write(
+          `logger: ${JSON.stringify(
```

Comment:
Why are we printing the logger out?

Reply:
Sorry, I don't know the previous intention. Perhaps @honzajavorek can clarify?

Reply:
https://github.com/apiaryio/dredd/pull/1497/files#r329025441 perhaps I'm confused, but seems this wasn't there before.

![Screenshot 2019-09-27 at 12 20 23](https://user-images.githubusercontent.com/44164/65765895-56539780-e121-11e9-818a-312ef2440b99.png)

Reply:
That was some left-overs, as turned out. Removed in 7b15907.